module/resource-tracker/resource-tracker-manager.js

```diff
@@ -1,27 +1,10 @@
-import * as Settings from '../../lib/miscellaneous-settings.js'
 import { arrayToObject, objectToArray } from '../../lib/utilities.js'
-import { ResourceTrackerEditor } from './resource-tracker-editor.js'
```

Comment:
`ResourceTrackerEditor` is now failing to resolve from within the resource tracker manager settings application.